src/sgame/components/RocketpodComponent.cpp

```diff
@@ -3,11 +3,11 @@
 
 #include <glm/geometric.hpp>
 
-constexpr float ATTACK_RANGE         = (float)ROCKETPOD_RANGE; // cgame needs to know this.
-constexpr int   ATTACK_PERIOD        = ROCKETPOD_ATTACK_PERIOD; // cgame needs to know this.
-constexpr int   SHUTTER_OPEN_TIME    = 1000;
-constexpr int   TARGET_SEARCH_PERIOD = 500;
-constexpr int   LOCKON_TIME          = 500;
+static const float &ATTACK_RANGE        = ROCKETPOD_RANGE; // cgame needs to know this.
```

Comment:
This reference stuff is kinda ugly; how about just replacing ATTACK_RANGE with ROCKETPOD_RANGE everywhere instead of proxying through another constant?

Or even better, don't put it in a global header at all; define the constant in this file only.

Reply:
> Or even better, don't put it in a global header at all; define the constant in this file only.

That can't be done, cgame needs to be aware of the value.

> How about just replacing ATTACK_RANGE with ROCKETPOD_RANGE everywhere […]?

I could, although then the constants are in different places.

Reply:
Let's go with replacing ATTACK_RANGE with ROCKETPOD_RANGE everywhere, it sucks to have multiple names for the same constant anyway.